### PR TITLE
Fix `AiidaComputerSetup` widget.

### PR DESCRIPTION
**Pull request by yakutovicha**
The widget wasn't working correctly as during the computer configuration
the value for the ssh port was of type "str". The issue is fixed
by converting the value type to "int".

Additionally, the `_configure_computer` function got slightly simplified by using
the `Computer.configure()` method provided by AiiDA.

### CODE REVIEW

File: aiidalab_widgets_base/computational_resources.py [hunk 1]
```diff
@@ -728,13 +732,8 @@ def _configure_computer(self):
...
         elif "proxyjump" in sshcfg:
             authparams["proxy_jump"] = sshcfg["proxyjump"]
-        aiidauser = orm.User.objects.get_default()
 
-        authinfo = orm.AuthInfo(
-            computer=orm.Computer.objects.get(label=self.label.value), user=aiidauser
-        )
-        authinfo.set_auth_params(authparams)
-        authinfo.store()
+        computer.configure(**authparams)
```
Comment (csadorf) [inline aiidalab_widgets_base/computational_resources.py:736]: So this function does what the previous code did w.r.t to `authinfo`?

**Review by csadorf — COMMENTED** [review]
It seems that the PR description does not really match/motivate the proposed change here. Can you clarify that? Otherwise it looks good to me.

File: aiidalab_widgets_base/computational_resources.py (hunk 1 above)
Comment (yakutovicha) [reply aiidalab_widgets_base/computational_resources.py:736]: Well, here I incorporated a function provided by AiiDA that does exactly the same. Just a little simplification.

I will now modify the PR description to better describe what has happened here.

Comment (yakutovicha) [reply aiidalab_widgets_base/computational_resources.py:736]: done.

Comment (csadorf) [reply aiidalab_widgets_base/computational_resources.py:736]: Thanks!

**Review by csadorf — APPROVED** [review]
Good to go! 🚢

**Pull request merged**